pkgs/os-specific/linux/kernel/linux-minipli.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, hostPlatform, fetchFromGitHub, perl, buildLinux, ... } @ args:
+
+with stdenv.lib;
+
+let
+  version = "4.9.40";
+  sha256 = "1rv345hciwc1mjz6r8wx8f0fqd4mkc84q36nna7xq1rlgml8m933";
+in
+
+import ./generic.nix (args // {
+  inherit version;
+
+  extraMeta.branch = intersperse "." (take 2 (splitString "." version));
```

Comment:
For the moment it would. But as soon as the kernel switches to the next LTS version, this would need to change. I believe my solution is safer.